scripts/clear-tile-cache

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+BUCKET_NAME=$(docker-compose run --rm tilegarden bash -c 'echo -n $PFB_TILEGARDEN_CACHE_BUCKET')
```

Comment:
Do you think it's worth allowing the user to set this bucket themselves, without having to edit `common.yml`? e.g.:

```bash
if [[ -z "${BUCKET_NAME}" ]]; then
    BUCKET_NAME=$(docker-compose run --rm tilegarden bash -c 'echo -n $PFB_TILEGARDEN_CACHE_BUCKET')
fi
```

I agree with your comment in the PR body that it's hard to think of cases where this script would be useful other than clearing the preconfigured dev tile cache, so I leave the decision up to you.

Reply:
Yeah, on reflection that's probably worth having.  It might not be useful, but then again it might, in which case it's a lot easier to write it in now than muddle through without it later.